docs/api/doc-blocks/doc-block-source.mdx

```diff
@@ -102,12 +102,14 @@ Determines if the snippet is rendered in dark mode.
   Determines if [decorators](../../writing-stories/decorators.mdx) are rendered in the source code snippet.
 </IfRenderer>
 
-### `format`
+### `format` (deprecated - will be removed in Storybook 9)
 
 Type: `boolean | 'dedent' | BuiltInParserName`
 
 Default: `parameters.docs.source.format` or `true`
 
+Deprecated: Will be removed in Storybook 9. Please use `docs.source.transform` instead and use your own formatter
```

Comment:
I think an example of how to do this with prettier here would come a long way.

Reply:
Agree